Tests/StreamChatTests/Repositories/AuthenticationRepository_Tests.swift

```diff
@@ -549,6 +590,9 @@ final class AuthenticationRepository_Tests: XCTestCase {
         let testError = TestError()
         connectionRepository.connectResult = .failure(testError)
 
+        // API Result
+        apiClient.test_mockResponseResult(Result<GuestUserTokenPayload, Error>.success(GuestUserTokenPayload(user: CurrentUserPayload.dummy(userId: "", role: .user), token: apiToken)))
```

Comment:
We should format this one for better readability. Wonder why this was not done on the pre-commit hook?